style: Add GUT logo

punfil opened the pull request:
![obraz](https://github.com/museum-engineering-project/koillectionPG/assets/77413489/ddd268ff-747c-4130-b1b9-64cde0b7df7d)
![obraz](https://github.com/museum-engineering-project/koillectionPG/assets/77413489/9de6857e-111b-4cd8-aca9-fb01e3d43058)
![obraz](https://github.com/museum-engineering-project/koillectionPG/assets/77413489/8011d431-60cc-444a-b7f7-ed6c8f663e4c)

Documentation:
https://pg.edu.pl/dzial-promocji/marka-pg/logotyp
https://pg.edu.pl/documents/10607/73c49777-8161-4cb6-a9f7-efd9c9acea3c

punfil committed:
style: Use GUT logo and ETI museum name

Replace standard Koillection logo with GUT logo.
Replace Koillection header with ETI museum.

Signed-off-by: Wojciech Panfil <wojtek.panfil@hotmail.com>

punfil committed:
readme: Add information about fork

Signed-off-by: punfil <wojtek.panfil@hotmail.com>

punfil commented on templates/layout.html.twig:
```diff
@@ -8,8 +8,8 @@
-                             aria-label="Logo" height="45" width="45">
-                        <span>Koillection</span>
+                             aria-label="Logo" height="45" width="60">
+                        <span>Muzeum ETI</span>
```
TODO: It remains the same in different languages.
Can we put `if` here in Twig or translate?

Lenetis replied:
It is possible to translate this using something like `<span>{{ 'global.eti-museum'|trans }}</span>` and adding the appropriate translation to the translation files.
Keep in mind that languages without that new translation will display just the untranslated translation key instead, so it would be best to add that translation to all languages currently supported by Koillection.

punfil replied:
Done.
Added appropriate translation to files for all languages.

punfil committed:
template: Fix improper image handling

MS Edge improperly displayed the image as it was reported as image, but actually it was image/svg

Signed-off-by: punfil <wojtek.panfil@hotmail.com>
